tools/clang/unittests/HLSL/ExecutionTest.cpp

```diff
@@ -4058,7 +4058,7 @@ void ExecutionTest::WaveIntrinsicsActivePrefixTest(
         std::vector<T2> outputList(waveData->size());
         // sort inputList and masklist by lane id. input for each lane can be computed for its group index
         for (size_t j = 0, end = waveData->size(); j < end; ++j) {
-          int laneID = waveData->at(j)->laneIndex;
+          unsigned laneID = waveData->at(j)->laneIndex;
           // ensure that each lane ID is unique and within the range
           VERIFY_IS_TRUE(0 <= laneID && laneID < waveData->size());
```

Comment:
Of course, now `0 <= laneID` is `true` by definition since `laneID` is `unsigned`.